.rubocop.yml

```diff
@@ -96,7 +96,7 @@ Metrics/BlockLength:
     - "app/models/concerns/indexable.rb"
 
 Metrics/ClassLength:
-  Max: 156 # Default 100
+  Max: 167 # Default 100
```

Comment:
I want to resist us raising these limits. They should gradually trend downwards over time. Can we extract the resetting callbacks to a separate module?

Reply:
Good idea - I have moved these resetting callbacks into the Resettable module